src/vue/backend/src/views/GuidedSetupConnectApi.vue

```diff
@@ -0,0 +1,84 @@
+<template>
+    <GuidedSetupContentCard
+        cardTitle="Connect Your IDX Broker Account"
+        :steps="guidedSetupSteps"
+        :relatedLinks="links"
+        @back-step="goBackStep"
+        @skip-step="goSkipStep"
+        @continue="goContinue">
+        <template v-slot:description>
+            <p>This step is optional. A sentence or two about why you should connect IMPress for IDX Broker to your IDX Broker account. Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
+        </template>
+        <template v-slot:controls>
+            <ApiKey :error="error" :loading="loading" :success="success"/>
+        </template>
+    </GuidedSetupContentCard>
+</template>
+
+<script>
+import { mapState, mapActions } from 'vuex'
+import GuidedSetupContentCard from '@/templates/GuidedSetupContentCard.vue'
+import ApiKey from '@/components/ApiKey.vue'
+export default {
+    name: 'GuidedSetupConnectApi',
+    components: {
+        ApiKey,
+        GuidedSetupContentCard
+    },
+    data () {
+        return {
+            error: false,
+            loading: false,
+            success: false,
+            links: [
+                {
+                    text: 'Where can I find my API key?',
+                    href: '#where'
+                },
+                {
+                    text: 'IDX Broker Middleware',
+                    href: 'https://middleware.idxbroker.com/mgmt/'
+                },
+                {
+                    text: 'Sign up for IDX Broker',
+                    href: '#signUp'
+                }
+            ]
+        }
+    },
+    computed: {
+        ...mapState({
+            guidedSetupSteps: state => state.general.guidedSetupSteps
+        })
+    },
+    methods: {
+        ...mapActions({
+            generalSettingsStateChange: 'general/generalSettingsStateChange',
+            promiseWithTimeout: 'general/promiseWithTimeout'
+        }),
+        async apiCall () {
+            this.loading = true
+            await this.promiseWithTimeout()
+            this.loading = false
+            this.success = true
```

Comment:
Does not have success state as outlined in the design, moves from loading to next page.
![Screen Shot 2021-02-25 at 8 50 38 AM](https://user-images.githubusercontent.com/25048313/109187226-8c49e080-7746-11eb-9194-6c4b58fa84b4.png)